R/module_init_data.R

```diff
@@ -126,11 +126,18 @@ srv_init_data <- function(id, data) {
   vapply(
     datanames,
     function(dataname, datasets) {
-      hash <- rlang::hash(data[[dataname]])
+      x <- data[[dataname]]
+
+      code <- if (is.function(x) && !is.primitive(x)) {
+        x <- deparse1(x)
```

Comment:
@averissimo why do you need this line? `x` is not used later on

Reply:
`x` is overwritten here (L132) and then it's reused on L140 (`deparse1(rlang::hash(x)),`)

Where the hash is being calculated for the `stopifnot` expression

Reply:
ah, yeah it is